Packs/QRadar/Playbooks/playbook-QRadar_Indicator_Hunting_V2.yml

```diff
@@ -57,6 +61,11 @@ tasks:
     note: false
     timertriggers: []
     ignoreworker: false
+    continueonerrortype: ""
+    skipunavailable: false
```

Comment:
should it be true?

Reply:
Same Here - why this should be also set to true if this is a title?